insecure/integration/functional/test/gossipsub/rpc_inspector/validation_inspector_test.go

```diff
@@ -1018,6 +1019,21 @@ func TestValidationInspector_InspectRpcPublishMessages(t *testing.T) {
 // The victim node is configured to use the GossipSubInspector to detect spam and the scoring system to mitigate spam.
 // The test ensures that the victim node is disconnected from the spammer node on the GossipSub mesh after the spam detection is triggered.
 func TestGossipSubSpamMitigationIntegration(t *testing.T) {
+	t.Run("gossipsub spam mitigation invalid grafts", func(t *testing.T) {
+		testGossipSubSpamMitigationIntegration(t, p2pmsg.CtrlMsgGraft)
+	})
+	t.Run("gossipsub spam mitigation invalid prunes", func(t *testing.T) {
+		testGossipSubSpamMitigationIntegration(t, p2pmsg.CtrlMsgPrune)
+	})
```

Comment:
Please also extend this test to `iHave` as well. Effectively, unknown, malformed and duplicate topics for `iHave` messages are also subsumed as spam attacks.

Reply:
https://github.com/onflow/flow-go/pull/5323/commits/82359ca75788765b717006b26d978ec782291dbc